### DISCUSSION TITLE
Support Mouse/Trackpad on iPadOS

### DISCUSSION THREAD

**Discussion by necopinus** [GitJournal/GitJournal#790] [Feature Requests]
## Environment

iPadOS

## Description

Currently, GitJournal does not appear to support mouse/trackpad input on iPadOS. Mostly, this is cosmetic (cursor doesn’t change when moving over various UI elements), but one place where it’s really apparent is that two-finger scrolling doesn’t work (to scroll any view using a mouse and trackpad requires click-and-drag, rather than using gestures).

Some potentially relevant Apple links:

- https://support.apple.com/en-us/HT211008
- https://developer.apple.com/design/human-interface-guidelines/ios/user-interaction/pointers/
- https://developer.apple.com/documentation/bundleresources/information_property_list/uiapplicationsupportsindirectinputevents

**Comment by vHanda** [comment]
Thanks for reporting this. GitJournal is using a relatively new framework (Flutter) which allows me to target ios/Android/desktop/web while sharing large portions of the code. Flutter is however still quite young.

Let me follow up with the Flutter developers to see if we can get support for mouse / trackpad input. Especially since this would also be relevant in the case of the desktop app. I'll keep you updated.

**Comment by keelancook** [comment]
This may or may not be related (I'm not a developer, just a happy user), but I run into issues with mouse navigation using the Android app on my Chromebooks. When I use the touch interface, all is fine. However, when I plug into an external monitor, the mouse works but scroll is almost unusable.
